src/native/libs/System.JavaScript/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+# System.JavaScript
+
+Is TypeScript project that is compiled via `src\native\libs\rollup.config.js` into Emscripten JS library.
+And a native library compiled by CMake into `System.JavaScript.a` as part of `/src/native/libs/CMakeLists.txt`
+
+## System.JavaScript.ts -> dotnet.native.js
+
+This is root of **Emscripten library** that would become part of `dotnet.runtime.js`
+It implements native parts of PAL for the VM/runtime.
```

Comment:
> It implements native parts of PAL for the VM/runtime.

What parts of the VM/runtime?

src/native/libs contained PAL for the managed libraries so far, roughly split by namespace. For example, `System.Security.Cryptography.Native` has PAL for the System.Security.Cryptography namespace.

I see this contains `SystemJS_RandomBytes` method, so this looks like a PAL for System namespace. Should it be called `System.Native.Browser` - to follow the pattern established by `System.Security.Cryptography.Native.Apple` and `System.Security.Cryptography.Native.Android`?

Reply:
OK, thanks!